distribution/changelog.txt

```diff
@@ -36,9 +36,7 @@
 - Fix: [#19950] Mine train block brake supports drawn incorrectly.
 - Fix: [#19955] Mine Train Roller Coaster has incorrect supports on the sloped left small turn (original bug).
 - Fix: [#19987] [Plugin] ‘SetCheatAction’ has wrong ID in plugin API.
-- Fix: [#19991] Crash using cut-away view.
 - Fix: [#20016] The group box for small scenery details in the Tile Inspector window has unused empty space.
-- Fix: [#20018] Shops not calculating up-keep cost.
```

Comment:
This is actually in 0.4.4 so shouldn't this be still in there? The regression was caused by #19683

Reply:
My bad, I thought the improved speed wasn't in the last release already.